src/test/java/Homework18.java

```diff
@@ -0,0 +1,58 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework18 extends BaseTest {
+
+    @Test
+    public static void playSong() {
+
+        driver.get(url);
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+
+        // log into Koel
+        setCssTextBox("input[type='email']", "henry.carter@testpro.io");
+        setCssTextBox("input[type='password']", "password");
+
+        clickCssButton("button[type='submit']");
+
+        // click play next song
+        // causes the player control buttons to become visible, but there is
+        // no response from the player otherwise. I even attempting the
+        // following line twice.... Please advise. The video for the following
+        // lecture is terrible.
+        clickXpathButton("//*[@id='mainFooter']/div[1]/i[2]");
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**